src/routes/pages/Home.tsx

```diff
@@ -0,0 +1,70 @@
+import { Category } from '@/components/common/Category'
+import Playlist, { PlaylistType } from '@/components/playlist/Playlist';
+import { useHeaderStore } from '@/stores/header'
+import { useFetchPlaylists } from '@/hooks/useFetchPlaylists'
+import { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import InfiniteScroll from '@/components/infiniteScroll/InfiniteScroll'
+
+export default function Home() {
+  const setTitle = useHeaderStore(state => state.setTitle)
+  const navigate = useNavigate()
+  const [selectedCategories, setSelectedCategories] = useState<string[]>([
+    '전체'
+  ])
+
+  useEffect(() => {
+    setTitle('Home')
+  }, [setTitle])
+
+  const { data } = useFetchPlaylists(false)
+
+  const handlePlaylistClick = (playlist: PlaylistType | undefined | null) => {
+    if (!playlist) {
+      return;
+    }
```

Comment:
P3;

아래에 실제 사용되는 부분 보면 pl이 옵셔널이 아닌 것 같은데요.

```tsx
      {filteredAndSortedData &&
        filteredAndSortedData.map(pl => (
          <Playlist
            key={pl.id}
            playlist={pl} // pl은 옵셔널이 아님
            onClick={() => handlePlaylistClick(pl)}
          />
          ))}
```

undefined, null 케이스를 대응 안 해줘도 되는 것 아닌가요?
```suggestion
  const handlePlaylistClick = (playlist: PlaylistType) => {
```

Reply:
수정하였습니다!